addons/account/models/account_move.py

```diff
@@ -254,7 +254,7 @@ def _sequence_fixed_regex(self):
         compute='_compute_suitable_journal_ids',
     )
     highest_name = fields.Char(compute='_compute_highest_name')
-    made_sequence_hole = fields.Boolean(compute='_compute_made_sequence_hole', search="_search_made_sequence_hole")
+    made_sequence_gap = fields.Boolean(compute='_compute_made_sequence_gap', store=True)
```

Comment:
Way too long IMO, the compute function is shorter :shrug:

Reply:
indeed :D My point is just that the meaning and computation of this field isn't trivial and should be documented. You can propose shorter if you want. It could also be moved as docstring of the compute function (ok that doesn't change the ratio, but you get my point)

Reply:
Then everything should be fixed now then! (or I didn't understand some of your comments :smile:)